docs/core-concepts/core-sdk/cli/dotnet-new.md

```diff
@@ -0,0 +1,44 @@
+dotnet-new
+==========
+
+## NAME
+dotnet-new -- Create a new .NET Core project
+
+## SYNOPSIS
+dotnet-new [--type] [--lang]
+
+## DESCRIPTION
+The new command provides a convenient way to initalize a valid .NET Core project and sample source code to try out the CLI toolset. 
+
+This command is invoked in the context of a directory. When invoked, the command will result in two main artifacts being dropped to the directory: 
```

Comment:
It is the same but it's an easier word. See here for examples of people lost with the term artifact: :smile: 
http://stackoverflow.com/questions/25619927/what-is-the-meaning-of-artifact-in-software-testing
http://stackoverflow.com/questions/2487485/what-is-a-maven-artifact
http://stackoverflow.com/questions/7130282/what-is-an-artifact